<?xml version="1.0" encoding="utf-8"?>

<launch>

  <arg name="map_frame" default="/map" />
  <arg name="cad_frame" default="/marker2" />
  <arg name="scan_topic" default="/corrected_scan" />
  <arg name="distance_threshold" default="0.2" />
  <arg name="discrete_color" default="false" />

  <!-- changes node -->
  <node name="cpt_changes"
        pkg="cpt_changes"
        type="changes_ros_node"
        output="screen"
        respawn="true">
    <param name="off_model" value="$(find cpt_changes)/resources/room.off" />
    <param name="stl_model" value="$(find cpt_changes)/resources/model.dae" /> 
    <param name="map_frame" value="$(arg map_frame)" />
    <param name="cad_frame" value="$(arg cad_frame)" />
    <param name="scan_topic" value="$(arg scan_topic)" />
    <param name="distance_threshold" value="$(arg distance_threshold)" />
    <param name="discrete_color" value="$(arg discrete_color" />

  </node>
</launch>
